templates/books-show.html

```diff
@@ -0,0 +1,36 @@
+<h2>{{bsc.book.title}}</h2>
+<hr> 
+<div class="row">
+	<div class="col-xs-6">
+		<img ng-src="{{bsc.book.image}}" class="img-responsive">
+	</div>
+	<div class="col-xs-6">
+		<p><strong>Author:</strong> {{bsc.book.author}}</p>
+		<p><strong>Release Date:</strong> {{bsc.book.releaseDate}}</p>
+		<br>
+
+		<!-- buttons to edit/delete -->
+		<a href="#" ng-click="editForm=!editForm" class="btn btn-default">Edit <span class="glyphicon glyphicon-pencil"></span></a>&nbsp;
+		<a href="#" ng-click="bsc.deleteBook(bsc.book)" class="btn btn-default">Delete <span class="glyphicon glyphicon-trash"></span></a>
+
+		<!-- edit form -->
+		<div ng-show="editForm">
+			<br>
+	    <form ng-show="editForm" ng-submit="bsc.updateBook(bsc.book)">
+	      <div class="form-group">
+	        <input type="text" ng-model="bsc.book.title" class="form-control" placeholder="Title">
```

Comment:
Use labels rather than placeholders to describe what goes in inputs. See https://www.nngroup.com/articles/form-design-placeholders/